### DISCUSSION TITLE
Welcome to great_expectations Discussions!

### DISCUSSION THREAD

**Discussion by kyleaton** [great-expectations/great_expectations#4099] [Announcements / Releases]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

In order to stay organized, here are the different categories of our Discussions channel:
* Announcements / Releases
  * This is channel will include product updates, events, and community celebrations.
* Community Use Cases
  * This is a place for the community to show off their use case of Great Expectations. 
* Integrations 
   * Share integrations you've created or request integrations you would like for Great Expectations.
* RFC (Request For Comment)
   * We will be posting items in development that we are looking for feedback on. 
* General
   *For discussion about anything and everything (Use Support for troubleshooting). Have fun with it! 
* Ideas
* Support

If you would like to see any changes to our GitHub discussions. Feel free to reach out to @kyleaton here or on [our slack](https://greatexpectations.io/slack)

**Comment by sungeunyuk** [comment]
hi
i use hadoop eco system a lot. 
why not support impala , hive etc...?
